drivers/quota/projectquota.go

```diff
@@ -222,6 +223,32 @@ func (q *Control) GetQuota(targetPath string, quota *Quota) error {
 	return nil
 }
 
+// GetDiskUsage - get the current disk usage of a directory that was configured with SetQuota
+func (q *Control) GetDiskUsage(targetPath string, usage *directory.DiskUsage) error {
+	projectID, ok := q.quotas[targetPath]
+	if !ok {
+		return fmt.Errorf("quota not found for path : %s", targetPath)
```

Comment:
This should be a defined error constant, I believe.

Reply:
I've refactored to not duplicate this code